vhdbuilder/packer/components.json

```diff
@@ -218,7 +218,7 @@
       ],
       "multiArchVersions": [
         "1.10.5",
-        "1.10.5.3"
```

Comment:
the version we are currently using is 1.10.5 or 1.10.5.3?

Reply:
```
{{- else if eq .component "cilium-agent" -}}
1.10.5
{{- else if eq .component "cilium-operator" -}}
1.10.5
```